### DISCUSSION TITLE
怎么显示本地视频的预览

### DISCUSSION THREAD

**Discussion by ZhouNLP** [AlistGo/alist#646] [Q&A]
Ubuntu20.04

用rclone挂载本地后再添加到alist，能解决很多网盘API限制问题，但唯一的不足是本地目录不能显示图片及视频预览

**Comment by xhofe** [comment]
预览图由对应存储提供，本地未提供。

**Comment by Misaka-Mikoto-Tech** [comment]
昨天我安装了最新版的 Alist，一开始本地图片预览竟然可以显示了，但我设置了一通以后它又不显示了，我现在怎么都搞不出预览了，奇怪
